src/storage/storage_structure/lists/lists_update_iterator.cpp

```diff
@@ -91,7 +91,7 @@ void ListsUpdateIterator::slideListsIfNecessary(uint64_t endNodeOffsetInclusive)
                 const unordered_set<uint64_t> deletedRelOffsetsInList;
                 lists->fillInMemListsFromPersistentStore(nodeOffsetToSlide,
                     lists->getNumElementsFromListHeader(nodeOffsetToSlide), inMemList,
-                    deletedRelOffsetsInList);
+                    deletedRelOffsetsInList, nullptr /* updatedPersistentListOffsets */);
```

Comment:
Maybe make this an optional argument with default value of nullptr?

Reply:
changed the function signature to make `updatedPersistentListOffsets ` optional